### DISCUSSION TITLE
Vite/Webpack/Esbuild plugin for Markdoc

### DISCUSSION THREAD

**Discussion by wobsoriano** [markdoc/markdoc#22] [Show and tell]
Made a plugin that enables you to import markdown files on your Vite/Webpack projects!

https://github.com/wobsoriano/unplugin-markdoc

Sample with Vite:

```ts
// vite.config.ts
import { defineConfig } from 'vite'
import MarkdocPlugin from 'unplugin-markdoc/vite'

export default defineConfig({
  plugins: [MarkdocPlugin({
    nodes: {...},
    tags: {...},
    ...
  })]
});
```

```ts
import content from './contents/doc.md'
import Markdoc from '@markdoc/markdoc'

const html = Markdoc.renderers.html(content)
```

**Comment by mfix-stripe** [comment]
@wobsoriano I dig it!

**Reply by mfix-stripe** [reply]
@wobsoriano if you want, add `markdoc` to your repo's "Topics" for it to show up here: https://github.com/topics/markdoc

<img width="312" alt="image" src="https://user-images.githubusercontent.com/62121649/168187786-7c928b98-34fd-45db-a4b8-c47b4a68f0f6.png">

**Reply by wobsoriano** [reply]
> @wobsoriano if you want, add `markdoc` to your repo's "Topics" for it to show up here: https://github.com/topics/markdoc
> 
> <img alt="image" width="312" src="https://user-images.githubusercontent.com/62121649/168187786-7c928b98-34fd-45db-a4b8-c47b4a68f0f6.png">

Alright thanks! Also working on Vue renderers and stuff right now 👌
